channels_api/routing.py

```diff
@@ -0,0 +1,67 @@
+from django.core.serializers.json import json
```

Comment:
I don't think this module is actually needed. I would expect the test code to just use the standard channels.auth decorators like `channel_session_user_from_http`